front50-sql/front50-sql.gradle

```diff
@@ -26,6 +26,8 @@ dependencies {
   testImplementation "org.testcontainers:postgresql"
   testImplementation "org.postgresql:postgresql"
 
+  testImplementation "com.h2database:h2"
```

Comment:
I think this is okay for now but we've trended away from h2 towards testcontainers for most of the SQL testing. I think the use case you have this here for is okay and makes sense (just initing storage services) but wouldn't want to see anything beyond that using h2.

Reply:
In that case, I'd like to see a comment to that nature here.

Reply:
Done!!